locale/fi/LC_MESSAGES/django.po

```diff
@@ -882,49 +887,93 @@ msgstr "Lisää uusia määräajan muutoksia"
 msgid "Deadline deviations"
 msgstr "Määräajan muutokset"
 
-#: deviations/templates/deviations/list_dl.html:27
+#: deviations/templates/deviations/list_dl.html:31
+#: deviations/templates/deviations/override.html:20
 msgid "Submitter"
 msgstr "Palauttaja"
 
-#: deviations/templates/deviations/list_dl.html:28
+#: deviations/templates/deviations/list_dl.html:32
+#: deviations/templates/deviations/override.html:21
 #: edit_course/templates/edit_course/edit_content.html:149
 #: exercise/templates/exercise/_user_results.html:81
 msgid "Exercise"
 msgstr "Tehtävä"
 
-#: deviations/templates/deviations/list_dl.html:29
+#: deviations/templates/deviations/list_dl.html:33
 msgid "Extra minutes"
 msgstr "Ylimääräiset minuutit"
 
-#: deviations/templates/deviations/list_dl.html:30
+#: deviations/templates/deviations/list_dl.html:34
 msgid "Without late penalty"
 msgstr "Ilman myöhästymissakkoa"
 
-#: deviations/templates/deviations/list_dl.html:31
+#: deviations/templates/deviations/list_dl.html:35
 #: edit_course/templates/edit_course/usertag_list.html:32
 msgid "Actions"
 msgstr "Toiminnot"
 
-#: deviations/templates/deviations/list_dl.html:51
+#: deviations/templates/deviations/list_dl.html:55
+#: deviations/templates/deviations/override.html:42
 msgid "No deadline deviations."
 msgstr "Ei yhtään määräajan muutosta."
 
-#: deviations/views.py:43
+#: deviations/templates/deviations/override.html:10
+msgid ""
+"Some of the deviations already existed! Please select the deviations you "
+"want to override!"
+msgstr "Osa määräajan muutoksista oli jo olemassa!"
+"Valitse ylikirjoitettavat muutokset!"
+
+#: deviations/templates/deviations/override.html:16
+msgid "Select deviations to be overridden"
+msgstr "Valitse ylikirjoitettavat muutokset"
+
+#: deviations/templates/deviations/override.html:22
+msgid "Extra time before (minutes)"
+msgstr "Lisä aika ennen (minuuteissa)"
+
+#: deviations/templates/deviations/override.html:23
+msgid "New extra time (minutes)"
+msgstr "Uusi lisäaika (minuuteissa)"
+
+#: deviations/templates/deviations/override.html:24
+msgid "Without late penalty before"
+msgstr "Ei myöhästymissakkoa ennen"
+
+#: deviations/templates/deviations/override.html:25
+msgid "Without late penalty after"
+msgstr "Ei myöhästymissakkoa jälkeen"
+
+#: deviations/templates/deviations/override.html:26
+msgid "Override"
+msgstr "Ylikirjoita"
+
+#: deviations/views.py:93
 msgid "You have to provide either minutes or a date in the future."
 msgstr "Sinun on määriteltävä joko minuutit tai päivämäärä."
 
-#: deviations/views.py:77
-#, python-brace-format
-msgid ""
-"Deadline deviation already exists for {user} in {exercise}!Remove it before "
-"trying to add a new one."
-msgstr ""
-"Määräajan muutos on jo olemassa opiskelijalle {user} tehtävässä {exercise}! "
-"Poista se ennen uuden muutokse lisäämistä."
+#: deviations/views.py:177
+msgid "Nothing removed!"
+msgstr "Mitään ei poistettu!"
+
+#: deviations/views.py:179
+msgid "Removed one deviation!"
```

Comment:
"Removed one deviation!", "Removed {number} deviations!"
You have two strings for the same thing, but singular and plural forms are separated. Plural forms should be implemented in the same string since gettext supports it. In the HTML template, you would use `{% blocktrans %}` with the `plural` section.

https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#blocktrans-template-tag

Reply:
In Python code, there are functions for doing the same: https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#pluralization